pkgs/tools/filesystems/duperemove/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchFromGitHub, libgcrypt
-, pkg-config, glib, linuxHeaders ? stdenv.cc.libc.linuxHeaders, sqlite }:
+, pkg-config, glib, linuxHeaders, sqlite }:
```

Comment:
Does that work?

Reply:
The default value was never used, so nothing has changed.